### PR TITLE
NFDIV-3118: Add common header and footer and update template versions

### PR DESCRIPTION
**Pull request by FaisalMoJ**
### Change description ###

Enter a description.

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/NFDIV-3118

**Before merging a pull request make sure that:**

- [X] tests have been updated / new tests has been added (if needed)
- [ ] README and other documentation has been updated / added (if needed)

**If this ticket will have any visible impact on users and is not behind a feature toggle, make sure that:**
- [ ] this ticket been reviewed by QA
- [ ] the user story been signed off by the PO

Note that bug fixes, dependency updates and technical tasks do not directly impact the user experience and can be merged without QA and PO review.

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/divorce/testutil/TestDataHelper.java [hunk 1]
```diff
@@ -25,36 +25,7 @@
...
-import uk.gov.hmcts.divorce.divorcecase.model.GeneralOrderJudgeOrLegalAdvisorType;
-import uk.gov.hmcts.divorce.divorcecase.model.GeneralParties;
-import uk.gov.hmcts.divorce.divorcecase.model.HelpWithFees;
-import uk.gov.hmcts.divorce.divorcecase.model.Jurisdiction;
-import uk.gov.hmcts.divorce.divorcecase.model.LanguagePreference;
-import uk.gov.hmcts.divorce.divorcecase.model.MarriageDetails;
-import uk.gov.hmcts.divorce.divorcecase.model.Solicitor;
-import uk.gov.hmcts.divorce.divorcecase.model.SolicitorService;
-import uk.gov.hmcts.divorce.divorcecase.model.UserRole;
+import uk.gov.hmcts.divorce.divorcecase.model.*;
```
Comment (amoriartyMOJ) [inline src/test/java/uk/gov/hmcts/divorce/testutil/TestDataHelper.java:28]: Looks like your IDE has used *, can we undo this change?

**Review by amoriartyMOJ — APPROVED** [review]

File: src/integrationTest/java/uk/gov/hmcts/divorce/caseworker/CaseworkerIssueApplicationIT.java [hunk 2]
```diff
@@ -1026,8 +1026,8 @@ void shouldIssueApplicationAndGenerateDocumentsWhenApplicant2IsRepresentedInJoin
 
         stubForDocAssemblyWith(AOS_COVER_LETTER_TEMPLATE_ID, "NFD_CP_Dummy_Template.docx");
         stubForDocAssemblyWith(DIVORCE_APPLICATION_TEMPLATE_ID, TEST_DIVORCE_APPLICATION_JOINT_TEMPLATE_ID);
-        stubForDocAssemblyWith(NOTICE_OF_PROCEEDING_TEMPLATE_ID, "NFD_Notice_Of_Proceedings_Joint_V2.docx");
-        stubForDocAssemblyWith(NOTICE_OF_PROCEEDING_TEMPLATE_ID, "FL-NFD-GOR-ENG-Notice-Of-Proceedings-AS1-V2.docx");
+        stubForDocAssemblyWith(NOTICE_OF_PROCEEDING_TEMPLATE_ID, "FL-NFD-GOR-ENG-Notice-Of-Proceedings-Joint-V3.docx");
```
Comment (amoriartyMOJ) [inline src/integrationTest/java/uk/gov/hmcts/divorce/caseworker/CaseworkerIssueApplicationIT.java:1029]: This is a new document name isn't it. So you could get away with no doing V3, as it's the first document of that name.

File: src/main/java/uk/gov/hmcts/divorce/document/content/NoticeOfProceedingContent.java [hunk 3]
```diff
@@ -140,30 +136,15 @@ public class NoticeOfProceedingContent {
     @Autowired
     private DocmosisTemplatesConfig config;
 
-    @Value("${court.locations.serviceCentre.serviceCentreName}")
```
Comment (amoriartyMOJ) [inline src/main/java/uk/gov/hmcts/divorce/document/content/NoticeOfProceedingContent.java:143]: I see a few more of these have been removed (centre name, service centre and phone number) do these need to be added to DocmosisCommonContent for the template to still work?

File: src/test/java/uk/gov/hmcts/divorce/document/content/NoticeOfProceedingContentTest.java [hunk 4]
```diff
@@ -264,17 +247,16 @@ public void shouldSuccessfullyApplyCivilPartnershipContentForNoticeOfProceedings
...
             .poBox("PO Box 13226")
             .town("Harlow")
             .postcode("CM20 9UG")
-            .phoneNumber("0300 303 0642")
             .build();
 
         when(commonContent.getPartner(caseData, caseData.getApplicant2(), ENGLISH)).thenReturn(CIVIL_PARTNER);
         when(holdingPeriodService.getDueDateFor(LocalDate.of(2021, 6, 18)))
             .thenReturn(LocalDate.of(2021, 11, 6));
+        when(docmosisCommonContent.getBasicDocmosisTemplateContent(caseData.getApplicant1().getLanguagePreference()))
```
Comment (amoriartyMOJ) [inline src/test/java/uk/gov/hmcts/divorce/document/content/NoticeOfProceedingContentTest.java:258]: Optional as I think it should have been done sooner but could you update the TestDataHelper.getBasicDocmosisTemplateContent method to include the ctsc vars?

File: src/test/java/uk/gov/hmcts/divorce/document/content/NoticeOfProceedingContentTest.java [hunk 5]
```diff
@@ -307,7 +289,6 @@ public void shouldSuccessfullyApplyCivilPartnershipContentForNoticeOfProceedings
                 entry(DIVORCE_OR_END_YOUR_CIVIL_PARTNERSHIP, APPLICATION_TO_END_YOUR_CIVIL_PARTNERSHIP),
                 entry(BEEN_MARRIED_OR_ENTERED_INTO_CIVIL_PARTNERSHIP, ENTERED_INTO_A_CIVIL_PARTNERSHIP_WITH),
                 entry(MARRIAGE_OR_CIVIL_PARTNER, CIVIL_PARTNERSHIP),
-                entry("ctscContactDetails", ctscContactDetails),
```
Comment (amoriartyMOJ) [inline src/test/java/uk/gov/hmcts/divorce/document/content/NoticeOfProceedingContentTest.java:310]: As per my previous comment, if you update the TestDataHelper.getBasicDocmosisTemplateContent  method to include ctsc, we can add this back and confirm they are set correctly?

**Review by amoriartyMOJ — APPROVED** [review]

**Review by amoriartyMOJ — APPROVED** [review]

**Comment by FaisalMoJ** [comment]
This is now covered by PR #2684
